### ISSUE TITLE
has this been tested as working?

### ISSUE THREAD

**Issue by skord** [kaimallea/csgo#4]
First off, you're an electric jesus for even attempting this, it's much appreciated. I want to spend as much time relaxing around Docker projects as I do working on them full time. 

That said, I can't seem to get the container running. Running: 

```
docker run -d --name csgo -p 27017:27017/udp -p 27017:27017 kmala/csgo -game csgo -console -usercon +game_type 0 +game_mode 0 +mapgroup mg_bomb +map de_dust2
```

Gives me no servers I can connect to. I'll dump the container logs at the the end of this issue. If you have gotten this to succesfully start and host games and I'm doing something stupid, let me know. If not, let me know and I'll be more than happy to throw some Docker know how into this project and the upstream project to get them working. 

```
./srcds_run: 32: ./srcds_run: pushd: not found
./srcds_run: 35: ./srcds_run: popd: not found
Server will auto-restart if there is a crash.
Using breakpad crash handler
[S_API FAIL] SteamAPI_Init() failed; SteamAPI_IsSteamRunning() failed.
[S_API FAIL] SteamAPI_Init() failed; unable to locate a running instance of Steam, or a local steamclient.dll.
LD_LIBRARY_PATH=/home/steam/csgo/bin:/home/steam/csgo:/home/steam/csgo/bin:
#Using breakpad minidump system
#
#Console initialized.
#Loading VPK file hashes for pure server operation.
#Loading VPK file hashes for pure server operation.
#Loading VPK file hashes for pure server operation.
#Loading VPK file hashes for pure server operation.
#Loading VPK file hashes for pure server operation.
#Loading VPK file hashes for pure server operation.
#Loading VPK file hashes for pure server operation.
#Loading VPK file hashes for pure server operation.
#Loading VPK file hashes for pure server operation.
#Game.dll loaded for "Counter-Strike: Global Offensive"
#CGameEventManager::AddListener: event 'server_pre_shutdown' unknown.
#CGameEventManager::AddListener: event 'game_newmap' unknown.
#CGameEventManager::AddListener: event 'finale_start' unknown.
#CGameEventManager::AddListener: event 'round_start' unknown.
#CGameEventManager::AddListener: event 'round_end' unknown.
#CGameEventManager::AddListener: event 'difficulty_changed' unknown.
#GameTypes: missing mapgroupsSP entry for game type/mode (custom/custom).
Failed to load gamerulescvars.txt, game rules cvars might not be reported to management tools.
Server is hibernating
No web api auth key specified - workshop downloads will be disabled.
maxplayers set to 64
Unknown command "cl_bobamt_vert"
Unknown command "cl_bobamt_lat"
Unknown command "cl_bob_lower_amt"
Unknown command "cl_viewmodel_shift_left_amt"
Unknown command "cl_viewmodel_shift_right_amt"
Unknown command "cl_teamid_min"
Unknown command "cl_teamid_max"
Unknown command "cl_teamid_overhead"
Unknown command "cl_teamid_overhead_maxdist"
Setting mapgroup to 'mg_bomb'
Network: IP 172.17.0.8 ---- Host_NewGame ----
Host_NewGame on map de_dust2
GameTypes: missing mapgroupsSP entry for game type/mode (custom/custom).
ConVarRef room_type doesn't point to an existing ConVar
bot_autodifficulty_threshold_high - 2.0
bot_autodifficulty_threshold_low - -6.0
bot_chatter - normal
bot_defer_to_human_goals - 0
bot_defer_to_human_items - 1
bot_difficulty - 1
bot_quota - 10
bot_quota_mode - normal
bot_allow_grenades - 1
bot_allow_pistols - 1
bot_allow_sub_machine_guns - 1
bot_allow_shotguns - 1
bot_allow_rifles - 1
bot_allow_snipers - 1
bot_allow_machine_guns - 1
mp_playercashawards - 1
cash_player_bomb_defused - 300
cash_player_bomb_planted - 300
cash_player_damage_hostage - -30
cash_player_interact_with_hostage - 150
cash_player_killed_enemy_default - 300
cash_player_killed_enemy_factor - 1
cash_player_killed_hostage - -1000
cash_player_killed_teammate - -3300
cash_player_rescued_hostage - 1000
cash_player_respawn_amount - 0
cash_player_get_killed - 0
mp_teamcashawards - 1
cash_team_elimination_bomb_map - 3250
cash_team_elimination_hostage_map_t - 1000
cash_team_elimination_hostage_map_ct - 2000
cash_team_hostage_alive - 0
cash_team_hostage_interaction - 500
cash_team_loser_bonus - 1400
cash_team_loser_bonus_consecutive_rounds - 500
cash_team_planted_bomb_but_defused - 800
cash_team_rescued_hostage - 0
cash_team_terrorist_win_bomb - 3500
cash_team_win_by_defusing_bomb - 3250
cash_team_win_by_hostage_rescue - 3500
cash_team_win_by_time_running_out_bomb - 3250
cash_team_win_by_time_running_out_hostage - 3250
mp_afterroundmoney - 0
mp_buytime - 90
mp_buy_anywhere - 0
mp_buy_during_immunity - 0
mp_startmoney - 800
mp_maxmoney - 16000
mp_death_drop_defuser - 1
mp_death_drop_grenade - 2
mp_death_drop_gun - 1
mp_deathcam_skippable - 1
mp_defuser_allocation - 0
mp_give_player_c4 - 1
mp_free_armor - 0
mp_molotovusedelay - 15.0
mp_weapons_allow_zeus - 1
mp_weapons_allow_map_placed - 0
mp_weapons_glow_on_ground - 0
mp_ct_default_melee - weapon_knife
mp_ct_default_secondary - weapon_hkp2000
mp_ct_default_primary - 
mp_ct_default_grenades - 
mp_t_default_melee - weapon_knife
mp_t_default_secondary - weapon_glock
mp_t_default_primary - 
mp_t_default_grenades - 
mp_buy_allow_grenades - 1
mp_freezetime - 6
mp_force_pick_time - 15
mp_halftime - 0
mp_match_can_clinch - 1
mp_maxrounds - 0
mp_roundtime - 5
mp_roundtime_defuse - 0
mp_roundtime_hostage - 0
[S_API FAIL] SteamAPI_Init() failed; SteamAPI_IsSteamRunning() failed.
Setting breakpad minidump AppID = 730
Forcing breakpad minidump interfaces to load
dlopen failed trying to load:
//.steam/sdk32/steamclient.so
with error:
//.steam/sdk32/steamclient.so: cannot open shared object file: No such file or directory
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
dlopen failed trying to load:
//.steam/sdk32/steamclient.so
with error:
//.steam/sdk32/steamclient.so: cannot open shared object file: No such file or directory
Looking up breakpad interfaces from steamclient
Calling BreakpadMiniDumpSystemInit
```

**Comment by sirtobey** [comment]
Any update on this?

**Comment by johnjelinek** [comment]
Update?

**Comment by hakabe** [comment]
Please use full path to install directory. Otherwise it's just nagging 'need to restart server for update' and nothing happens.
http://forums.steampowered.com/forums/showthread.php?t=2894570

**Comment by kaimallea** [comment]
Fixed by #6, tested and working
